### ISSUE TITLE
support litecoin testnet and/or bitcoin signet

### ISSUE THREAD

**Issue by mcudev** [trezor/trezor-suite#3568]
**Rationale**

I have been helping new users get setup, comfortable, and trained with their new Trezor T devices.

So far, the easiest option with Trezor Suite has been to use the existing Bitcoin Testnet to demonstrate to them how to send and receive coins in a limited risk environment, etc...

However, Bitcocin Testnet is notoriously unreliable and can lead to new user frustration.
Bitcoin signet was implemented with one of the use cases being: "You are an Instructor, and want to run a controlled Bitcoin network environment for teaching purposes." https://en.bitcoin.it/wiki/Signet
It would be great to have the ability to use Bitcoin signet with Trezor Suite.

In practice, Bitcoin signet is still pretty unreliable. As I write this, there has not been a block in nearly an hour. Litecoin testnet has in practice been much more reliable, and is great for user training due to the faster block times. Due to Litecoin being a very similar fork of Bitcoin, there is a great overlap in learning and practice. This dovetails nicely with getting p2wpkh for Litecoin as part of #3399.

Blockbook already supports Bitcoin signet and Litecoin Testnet. There would be some ongoing overhead for running these nodes (on the bright side, these test networks are low resource usage so far). The nodes would not necessarily need to be hosted by trezor, but would certainly make this even easier for people to use.

The firmware already supports signet via the existing support for Testnet: `trezorctl btc get-address -t wpkh -n "m/84h/1h/0h/0/0" -c Testnet`
The firmware already supports Litecoin testnet via existing support for "Litecoin Testnet": `trezorctl btc get-address -t wpkh -n "m/84h/1h/0h/0/0" -c 'Litecoin Testnet'`

Currently, Litecoin has not been updated to 0.21.0 or later and does not have the PSBT support to create unsigned transaction in the GUI and then send to Trezor for signing that Bitcoin signet does. This is another reason why support for both networks would be nice. Usually you get the latest features with Bitcoin as Litecoin usually adds them later (since it's a fork).

In summary, we'd need to stand up some blockbook nodes and then add support into Suite. The blockbook and firmware support already exists. This would be helpful for onboarding/training new users if there were the choice of quick, more reliable test networks, available within Trezor Suite.

**Comment by mcudev** [comment]
I forgot to mention that a bitcoin signet blockbook instance is running at https://signet-explorer.wakiyamap.dev/
That might help to test this. I don't know how reliable that instance is, I just found it while googling.

**Comment by yura-pakhuchiy** [comment]
I've opened issue with a bit more general version of this issue: https://github.com/trezor/trezor-suite/issues/3566

**Comment by mcudev** [comment]
I opened this and then I worked on it. I've changed my mind on requesting this. It's a lot of overhead that I myself would not want to support. So, why should I ask someone else to? I have ways of demonstrating concepts with a BTC regtest network and HWI if/when I need to.
